### DISCUSSION TITLE
Assembly/Kits

### DISCUSSION THREAD

**Discussion by nimish** [dukelec/cd_pnp#16] [General]
Hi, following up on my original post regarding a BOM or assembled kit. This project is really interesting and I'd like to build my own/use a pre assembled one myself.

In some places a for-sale version has been mentioned. Are there plans for it to be offered internationally?

**Comment by dukelec** [comment]
The machine is currently available for sale at the following link for you to take a look at:
https://www.alibaba.com/product-detail/_1600819959031.html
